.travis.yml

```diff
@@ -0,0 +1,35 @@
+notifications:
+  email: false
+
+branches:
+  only:
+    - master
+    - develop
+    - /^v\d+\.\d+\.\d+/
+
+language: node_js
+
+node_js:
+  - '10'
+  - '12'
+
+install:
+  - yarn install --frozen-lockfile --non-interactive
+
+jobs:
+  include:
+    - stage: Check TypeScript
+      script:
+        - yarn check:types
+    - stage: Lint
+      script:
+        - yarn lint
+    - stage: Check Formatting
+      script:
+        - yarn format:check
```

Comment:
Should this be `- yarn check:format`? [See here](https://github.com/FormidableLabs/next-urql/pull/30/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R18)!

Reply:
🤦‍♂ Yep, that'd explain why CI failed, good catch!